contracts/governance/utils/Votes.sol

```diff
@@ -0,0 +1,214 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../../utils/Context.sol";
+import "../../utils/Counters.sol";
+import "../../utils/Checkpoints.sol";
+import "./IVotes.sol";
```

Comment:
might be my OCD speaking here, But I'd prefer to have the imports more neatly ordered. IMO, `IVotes` should be after `draft-EIP721`.

Reply:
Resolved by `b171720c`